release-process.md

```diff
@@ -0,0 +1,106 @@
+# Release process
+
+This document describes our release process for open-source projects.
+
+## Contributors
+
+### New features, changes, and deprecations
+
+When adding new features (i.e., functions or classes), docstrings should properly
+indicate in which version the feature first appeared using the `.. versionadded::`
+directive.
+
+When changing the API,` .. versionchanged::` should be used. And deprecations should
+be indicated with `.. deprecated::`. Read
+[ploomber-core docs](https://ploomber-core.readthedocs.io/en/latest/deprecation.html)
+for details.
+
+
+### CHANGELOG
+
+Each repository contains a `CHANGELOG` file in the root directory. Each PR should
+contain a list of items, so we keep it up-to-date. Note that the `CHANGELOG` targets
+end-users (while `git log` targets the Ploomber development team); this implies
+that there might be changes that we don't include in the `CHANGELOG`, but they exist
+in the `git log`, for example changes to the CI configuration, new tests
+added/fixed.
+
+These are changes that we add to the `CHANGELOG`:
+
+- [Doc] Important documentation changes
```

Comment:
Those should rarely get in imo. It will probably also affect the way we use the CD.

Reply:
unless we map on prefix, still didn't get there.

Reply:
> unless we map on prefix, still didn't get there.

I'm not following

Reply:
We can rule out docs changes based on the [Doc] prefix. I think if only important ones get into the changelog it's fine and it should work fine.